insights/parsers/scheduler.py

```diff
@@ -1,17 +1,50 @@
-from .. import parser, get_active_lines, Parser
+"""
+Scheduler - file ``/sys/block/*/queue/scheduler``
+=================================================
+
+This parser parses the content from scheduler files. It stores available
+values and also current selection for every device.
+
+Sample content from schduler file:
+
+    noop deadline [cfq]
+
+Examples:
+    >>> type(scheduler_obj)
+    <class 'insights.parsers.scheduler.Scheduler'>
+    >>> scheduler_obj.data
+    {'sda': '[cfq]'}
+    >>> scheduler_obj.device
+    'sda'
+    >>> scheduler_obj.schedulers
+    ['noop', 'deadline', 'cfq']
+    >>> scheduler_obj.active_scheduler
+    'cfg'
+
+"""
+
 import re
+
 from insights.specs import Specs
+from .. import parser, get_active_lines, Parser
 
 
 @parser(Specs.scheduler)
 class Scheduler(Parser):
+    ACTIVE_SCHEDULER_PATTERN = re.compile(r'\[(.*)]')
```

Comment:
@skontar can you please add a minimal docstring here, since you have full docs in the top of the module.  Also can you please document the attributes you are adding to the class as well as the `data` attribute?